package.json

```diff
@@ -35,6 +35,7 @@
     "ember-prism": "^0.4.0",
     "ember-styleguide": "^2.5.0",
     "ember-tether": "^1.0.0",
+    "ember-truth-helpers": "^2.1.0",
```

Comment:
Hmm, it looks like we were already using `eq` (without having this dependency), can you double check that the app itself doesn't add a manual `eq`?

Reply:
It doesn't, I believe this was a dependency of another addon actually, it was in the lockfile